35/generated/mirror_heartbeat_config.html

```diff
@@ -330,6 +330,16 @@ <h4><a id="replication.policy.class"></a><a id="mirror_heartbeat_replication.pol
 </tbody></table>
 </li>
 <li>
+<h4><a id="replication.policy.internal.topic.separator.enabled"></a><a id="mirror_heartbeat_replication.policy.internal.topic.separator.enabled" href="#mirror_heartbeat_replication.policy.internal.topic.separator.enabled">replication.policy.internal.topic.separator.enabled</a></h4>
```

Comment:
Was this a forgotten documentation chunk that we forgot to add until now?

Reply:
No, https://issues.apache.org/jira/browse/KAFKA-15102 fixes for 3.5.2 and many other versions.